examples/abitest/tests/src/lib.rs

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright 2019 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+extern crate abitest_backend;
```

Comment:
I thought `extern crate` is not really necessary any more? https://doc.rust-lang.org/nightly/edition-guide/rust-2018/module-system/path-clarity.html